### ISSUE TITLE
iOS - Distance - Recent destination list animation is not smooth or seamless

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#40124]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.62-2
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Email or phone of affected tester (no customers):**
**Issue reported by:** Applause - Internal Team

## Action Performed:
Precondition:
- There is at least one address in Recent destination list
1. Launch New Expensify app
2. Go to FAB > Manual request > Distance
3. Tap Start
4. Repeat Step 3 several times

## Expected Result:
The Recent destination list should animate the same way it does in the production build

## Actual Result:
The Recent destination list animates different in staging build. Sometimes the list pops up from the center, sometimes it slides down from the top and the sliding down animation is not smooth
On production build, the list appears seamlessly without looking awkward

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/4a933555-b3b4-4dde-97c7-ab0cf3a5e742

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by lanitochka17** [comment]
@deetergp  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by lanitochka17** [comment]
We think that this bug might be related to #wave-collect - Release 1

**Comment by lanitochka17** [comment]
Production:

https://github.com/Expensify/App/assets/78819774/f778f8ed-ab9e-41b6-b5d0-90a8b05b3ea9

**Comment by deetergp** [comment]
@neil-marcellini Is this possibly related to https://github.com/Expensify/App/pull/39190?

**Comment by neil-marcellini** [comment]
Yeah I think it's what @ishpaul777 said. I really don't see a big problem here though. It looks fine to me. I'm going to remove the blocker label and close the issue. If anyone wants to re-open it we can do that, but I think we have much bigger problems elsewhere.
